### ISSUE TITLE
o-tabs: focus is taken off the website when activating a tab - ID: DAC_Name_Role_Value_Focus_Order_01

### ISSUE THREAD

**Issue by JakeChampion** [Financial-Times/origami#559]
# Issue
This is from page 197 of the [audit report](https://drive.google.com/file/d/17wq6b0CIX41RA4muBFDzUDxE0-qzHBIH/view)

Issue can be found on: <https://www.ft.com/__origami/service/build/v3/demo?component=o-tabs@6.0.0&demo=buttontabs&system_code=origami-repo-data&brand=master>

The tabs have been structured incorrectly with links being placed inside list items which have the role of tab. This issue may affect how screen reader users identify the tabs in the panel.
When users press enter to select a tab, the focus is taken of the page into the browser. This issue may affect keyboard alone and screen reader users. Keyboard alone users would have to tab back through the browser to get back to the tabs, whilst screen reader users may become disorientated and be unable to distinguish where the focus has been placed.

# Screen Reader user comments:

> “Although I could focus on and activate each of the tabs with the Screen Reader, using ‘F’ to jump to each of the tabs followed by ‘Enter’, I was not informed which tab was selected. This was confusing and also time consuming and may also confuse other Screen Reader users because extra steps are needed to verify which tab has been opened.
> Using NVDA I found that I could identify which tab had been chosen despite not being informed that a selection had been made.
> Using TalkBack and VoiceOver, I found that I could determine which tab had been selected when swiping in context.
> Ensuring that the Screen Reader informs the user which tab is selected at the point of activation should resolve confusion and make navigation swifter.
> Please note that this issue was partially present with NVDA, TalkBack and VoiceOver.”

# Keyboard only user comments:

> After I have activated the tab, my focus goes back into the main address bar, and I have to tab back into the page to select a different one. if this were a real content page users would not expect that to happen, they would expect their focus to continue where it was from otherwise, they would have to start from the top of the page again.

# Additional instances:
Additional instances of this issue exist on other pages throughout the website; wherever this issue occurs, they too will need to be resolved.
- URL: https://www.ft.com/__origami/service/build/v3/demo?component=o- tabs%406.0.0&demo=buttontabs-inverse&system_code=origami-repo-data&brand=master
   
- URL: https://www.ft.com/__origami/service/build/v3/demo?component=o- tabs%406.0.0&demo=buttontabs-big&system_code=origami-repo-data&brand=master

# Proposed Solution:
Change the links inside the list to buttons, this will enable screen readers to navigate and identify the tabs correctly.
Please refer to [ARIA Tab Panel Accessibility: A11y Support Series](https://www.deque.com/blog/a11y-support-series-part-1-aria-tab-panel-accessibility/), [Tabs - GOV.UK Design System](https://www.w3.org/TR/wai-aria-practices/examples/tabs/tabs-1/tabs.html) and [Example of Tabs with Manual Activation - W3.org](https://www.w3.org/TR/wai-aria-practices/examples/tabs/tabs-2/tabs.html)
